core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
Eg `urlForAction` is called > 50 times and there are two calls to getModule() just in this method. Of course it has not a super huge impact but it's simply faster and maybe even easier to read etc. Not sure what the problem with the property is? We can remove it of course, just don't understand the problem.